anyway/utilities.py

```diff
@@ -12,10 +12,23 @@
 
 import phonenumbers
 from dateutil.relativedelta import relativedelta
-from flask import Flask
+
+try:
+    from flask import Flask
+except ModuleNotFoundError:
+    pass
+
 from phonenumbers import NumberParseException
-from pyproj import Transformer
-from validate_email import validate_email
+
+try:
+    from pyproj import Transformer
+except ModuleNotFoundError:
+    pass
+
+try:
+    from validate_email import validate_email
+except ModuleNotFoundError:
+    pass
```

Comment:
IMO it's better to either move the imports to the functions that use them, or split this file to two modules based on available modules.

Reply:
to be handled in #1876